api/src/main/resources/metadata/Role_Privilege.xml

```diff
@@ -611,6 +611,8 @@
     <privilege  privilege="Manage TB Program"  description="Manage TB Program"  uuid="f9119a4f-db09-47c9-9371-2fc7e0e75ba4"/>
     <privilege  privilege="Manage EID Program"  description="Manage EID Program"  uuid="7494bc5c-44e8-4f45-bfa7-9b6e3f662fd0"/>
 
+    <privilege  privilege="Manage Procedures"  description="Allows the user to manage procedure orders"  uuid="ce981491-76d8-4925-b8d8-089352538a34"/>
```

Comment:
Are you adding these to any role

Reply:
No, the facility can choose which role to give this privilege